templates/cluster-template-clusterclass.yaml

```diff
@@ -699,7 +699,17 @@ metadata:
 spec:
   template:
     spec:
+      controlPlaneEndpoint:
```

Comment:
local test passed with these changes 
<pre>
Running kubectl apply --kubeconfig /var/folders/l3/8dxq0z892n992ffkmb33003r0000gn/T/e2e-kind779799432 -f -
stderr:

stdout:
kubeadmconfigtemplate.bootstrap.cluster.x-k8s.io/nutanix-quick-start-kcfg-0 created
clusterclass.cluster.x-k8s.io/nutanix-quick-start created
kubeadmcontrolplanetemplate.controlplane.cluster.x-k8s.io/nutanix-quick-start-kcpt created
nutanixclustertemplate.infrastructure.cluster.x-k8s.io/nutanix-quick-start-nct created
nutanixmachinetemplate.infrastructure.cluster.x-k8s.io/nutanix-quick-start-cp-nmt created
nutanixmachinetemplate.infrastructure.cluster.x-k8s.io/nutanix-quick-start-md-nmt created
configmap/quick-start-to7xov-pc-trusted-ca-bundle created
configmap/cni-quick-start-to7xov-crs-cni created
configmap/nutanix-ccm created
secret/quick-start-to7xov-pc-creds created
secret/nutanix-ccm-secret created
clusterresourceset.addons.cluster.x-k8s.io/quick-start-to7xov-crs-cni created
clusterresourceset.addons.cluster.x-k8s.io/nutanix-ccm-crs created
cluster.cluster.x-k8s.io/quick-start-to7xov created

  INFO: Waiting for the cluster infrastructure of cluster quick-start-dsyasz/quick-start-to7xov to be provisioned
  STEP: Waiting for cluster to enter the provisioned phase @ 05/02/24 11:27:41.078
  INFO: Waiting for control plane of cluster quick-start-dsyasz/quick-start-to7xov to be initialized
  INFO: Waiting for the first control plane machine managed by quick-start-dsyasz/quick-start-to7xov-dxd6k to be provisioned
  STEP: Waiting for one control plane node to exist @ 05/02/24 11:27:51.12
  INFO: Waiting for control plane of cluster quick-start-dsyasz/quick-start-to7xov to be ready
  INFO: Waiting for control plane quick-start-dsyasz/quick-start-to7xov-dxd6k to be ready (implies underlying nodes to be ready as well)
  STEP: Waiting for the control plane to be ready @ 05/02/24 11:29:21.212
  STEP: Checking all the control plane machines are in the expected failure domains @ 05/02/24 11:29:21.221
  INFO: Waiting for the machine deployments of cluster quick-start-dsyasz/quick-start-to7xov to be provisioned
  STEP: Waiting for the workload nodes to exist @ 05/02/24 11:29:21.287
  STEP: Checking all the machines controlled by quick-start-to7xov-md-0-2lsjj are in the "<None>" failure domain @ 05/02/24 11:29:21.31
  INFO: Waiting for the machine pools of cluster quick-start-dsyasz/quick-start-to7xov to be provisioned
  INFO: Calling PostMachinesProvisioned for cluster quick-start-dsyasz/quick-start-to7xov
  STEP: PASSED! @ 05/02/24 11:29:21.347
</pre>